src/notes.json

```diff
@@ -0,0 +1,146 @@
+[
+  {
+    "title": "Meeting Agenda",
+    "description": "Discuss quarterly sales projections and marketing strategies with the team.",
+    "createdAt": "2024-05-09T08:00:00Z",
+    "editedAt": "2024-05-09T08:00:00Z"
```

Comment:
interface Note {
    id: string;
    title: string;
    createdDate: Date;
    modifiedDate: Date;
    deletedDate?: Date | null;
    text: string;
    isPinned: boolean;
}

WDYT?

Reply:
IMO: description -> contents

Reply:
smth like this in progress [](https://github.com/wixplosives/NotesApp/pull/6/files)

Reply:
Added `id`, `isPinned` and optional `deletedAt`